src/scm/plams/interfaces/adfsuite/ase_calculator.py

```diff
@@ -156,33 +165,47 @@ class AMSCalculator(Calculator):
     # counters are a dict as a class variable. This is to support deepcopying/multiple instances with the same name
     _counter: Dict[str, int] = {}
 
-    def __new__(cls, settings=None, name="", amsworker=False, restart=True, molecule=None, extractors=[]):
+    def __new__(  # type: ignore
+        cls: Type[TSelf],
+        settings: Settings,
```

Comment:
This used to be optional but is now required - is this an intentional change?

Reply:
It was intentional, but should be reverted as discussed.